src/commands/ledger/download.ts

```diff
@@ -26,7 +26,7 @@ Validated 13,438 ledger events.
 
     initClient(flags.token)
 
-    const count = await downloadAndValidateLedger(flags.file)
```

Comment:
I'm thinking this caused the test to fail. Ketting by default already adds the token, so you don't need to pass it here.